OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -606,10 +606,7 @@ public Activity MoveToTarget(Actor self, Target target)
 
 		public Activity MoveIntoTarget(Actor self, Target target)
 		{
-			if (!Info.VTOL)
-				return new Land(self, target);
-
-			return new HeliLand(self, false);
+			return !Info.VTOL ? new Land(self, target) : new Land(self);
```

Comment:
Can we use the `self, target` version for both here?
If not, please drop the `!` and switch the cases to improve readability.